app/scripts/controllers/overview.js

```diff
@@ -933,6 +933,24 @@ function OverviewController($scope,
     });
   };
 
+  var updateServicesForOvms = function () {
+    if (!overview.offlineVirtualMachines || !state.allServices) {
+      return;
+    }
+    _.each(overview.offlineVirtualMachines, function(ovm) {
+      var ovmDomain = _.get(ovm, 'metadata.labels["kubevirt.io/domain"]');
```

Comment:
I find this way of getting `_.get` easier if some keys have special characters:

```js
_.get(ovm, ['metadata', 'labels', 'kubevirt.io/domain']);
```

Reply:
done